README.md

```diff
@@ -1 +1,49 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse Project
+
+This project is been carried out for the purpose of gaining mastery of the following:-
+1.	Repository Creation
+2.	Branch Creation
+3.	Linters Configuration
+4.	Creating of commit action with descriptive commit messages.
+5.	Working Remotely
+
+## Built With
+
+•	Major languages: Html and CSS,
+•	Technologies used are: Git and GitHub
```

Comment:
[OPTIONAL] Please use `-` instead of `•` to make bullet points so that each point would be on a separate line.